Civi/Share/Change.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Civi\Share;
+
+use Civi\Api4\ShareChange;
+
+class Change {
+
+  public const STATUS_LOCAL = 'LOCAL';
+
+  protected ?int $id = NULL;
+
+  protected string $type;
+
+  protected \DateTime $changedDate;
```

Comment:
It could be considered to use `\DateTimeInterface` here, for `$receivedDate`, and parameter type hints.